From afc2a95abd55053d224a97806b3417ada22f3ebe Mon Sep 17 00:00:00 2001
From: Eugeniy Paltsev <Eugeniy.Paltsev@synopsys.com>
Date: Fri, 22 Dec 2017 21:19:36 +0300
Subject: [PATCH] ARC: HSDK: USB: disable USB init

Disable default USB init as it break USB work in linux

Signed-off-by: Eugeniy Paltsev <Eugeniy.Paltsev@synopsys.com>
---
 board/synopsys/hsdk/hsdk.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/synopsys/hsdk/hsdk.c b/board/synopsys/hsdk/hsdk.c
index c20f00c2bf..b2f9ef3cae 100644
--- a/board/synopsys/hsdk/hsdk.c
+++ b/board/synopsys/hsdk/hsdk.c
@@ -62,7 +62,7 @@ int board_mmc_init(bd_t *bis)
 int board_early_init_r(void)
 {
 	/* Init USB to be able read environment from USB MSD */
-	usb_init();
+	/* usb_init(); */ /* Temporary disable init as it break usb in linux */
 
 	return 0;
 }
